### ISSUE TITLE
GIF export

### ISSUE THREAD

**Issue by ilkeralp** [IlyaKirillov/GoProject#36]
Is it possible to export gif files with comments. 

best 

ilker alp

**Comment by IlyaKirillov** [comment]
Hello. Do you mean to write down comments right in the picture?

**Comment by ilkeralp** [comment]
> Hello. Do you mean to write down comments right in the picture?

Yes, exactly. Sorry for my late response. I am trying to create my own notes from sgf files. GIF export helps alot, i can easyly create a pdf from gif files. But It will be very usefull if i can have the comments right in the picture.

**Comment by ilkeralp** [comment]
> > Hello. Do you mean to write down comments right in the picture?
> 
> Yes, exactly. Sorry for my late response. I am trying to create my own notes from sgf files. GIF export helps alot, i can easyly create a pdf from gif files. But It will be very usefull if i can have the comments right in the picture.

And maybe the comments could be below the gif image.
